app/controllers/best_answers_controller.rb

```diff
@@ -0,0 +1,24 @@
+class BestAnswersController < ApplicationController
+  before_action :require_owner
+
```

Comment:
This is really lean. Really like how you went about solving the BestAnswer problem!